lib/db/cms/services.ts

```diff
@@ -99,10 +99,36 @@ const deleteOfferingSection = async (
     }
 }
 
+const addOfferingEvent = async (
+    id: string,
+    section: string,
+    data: { description: string; title: string; img: string }
+): Promise<ServicesProps> => {
+    try {
+        const services = await getServices()
+        const updatedOfferings = [...services.offerings].map(offering => {
+            if (offering.section === section) {
+                const newEvents = [...offering.events]
```

Comment:
we can make this a sexy ternary 
`const updatedOfferings = offerings.map(offering =>
            offering.section === section
                ? { ...offering, events: [...offering.events, data] }
                : offering
        )`